lua/nvim-tree/git/init.lua

```diff
@@ -68,11 +68,18 @@ function M.get_project_root(cwd)
     return nil
   end
 
-  local project_root = git_utils.get_toplevel(cwd)
-  return project_root
+  if M.config.git.enable then
+    return git_utils.get_toplevel(cwd)
+  end
+
+  return nil
 end
 
 local function reload_tree_at(project_root)
+  if M.config.git.enable then
+    return nil
+  end
```

Comment:
@alex-courtis @yehy4 is this change correct ? It looks like it prevents git to reload when enabled.

Reply:
Sorry, that was a mistake. Thank you for catching it. Fixed in #1443.